data_loader/8_fakenews/fakenews_plugin.py

```diff
@@ -0,0 +1,167 @@
+"""Load data from the Squirro Fake News Training API"""
+
+import logging
+import hashlib
+import json
+import requests
+import urllib.parse
+from squirro.dataloader.data_source import DataSource
+
+log = logging.getLogger(__name__)
+
+
+class FakeNewsDataSource(DataSource):
+    """FakeNews Plugin for the sq_data_loader.
+
+    Features:
+    * Load a single section of the news
+    * Full and Incremental load
+    * Web UI and CLI Support
+    """
+
+    def __init__(self):
+        pass
+
+    def connect(self, inc_column=None, max_inc_value=None):
+        if self.args.reset:
+            log.info("Resetting key-value stores")
+            self.key_value_cache.clear()
+            self.key_value_store.clear()
```

Comment:
Is this needed? Doesn't the data loader reset option do this automatically?

Reply:
I have in the meantime decided that i will use two stages in the training video. First a very lean version where the plugin only support one news section and it will use the built in incr support. For that this is not needed.

But the 2nd stage i will support multiple sections with unique cursors, so hence i will need this.
I'll add the simple version to this PR as well.